tee-worker/ts-tests/integration-tests/scheduled_enclave.test.ts

```diff
@@ -0,0 +1,84 @@
+import { KeyObject } from 'crypto';
+import { step } from 'mocha-steps';
+import { initIntegrationTestContext } from './common/utils';
+import { getTeeShieldingKey } from './common/di-utils';
+import type { IntegrationTestContext, JsonRpcRequest } from './common/common-types';
+import { decodeRpcBytesAsString, sendRequest } from './common/call';
+import type { CorePrimitivesIdentity } from 'parachain-api';
+import { assert } from 'chai';
+import { createJsonRpcRequest, nextRequestId } from './common/helpers';
+
+describe('Scheduled Enclave', function () {
+    let context: IntegrationTestContext = undefined as any;
```

Comment:
why do we define the type and then cast it to `any`?

Reply:
good question 🤔 this one was copy-paste from another test. I just created [chore task](https://linear.app/litentry/issue/P-686/worker-ts-tests-cleanup) to cleanup our tests, as I've also seen some fixme's were copied from one test to another